src/cozmo/robot.py

```diff
@@ -483,14 +484,23 @@ def __init__(self, conn, robot_id, is_primary, **kw):
 
         self._action_dispatcher = self._action_dispatcher_factory(self)
 
+        #: :class:`cozmo.util.Speed` Speed of the left wheel
         self.left_wheel_speed = None
+        #: :class:`cozmo.util.Speed` Speed of the right wheel
         self.right_wheel_speed = None
-        self.light_height = None
+        #: :class:`cozmo.util.Distance` height of the lift from the ground (in :const:`MIN_LIFT_HEIGHT_MM` to :const:`MAX_LIFT_HEIGHT_MM` range)
+        self.lift_height = None
+        #: float: the current battery_voltage (not linear, but < 3.5 is low)
         self.battery_voltage = None
+        #: int: the ID of the object currently being carried (-1 if none)
         self.carrying_object_id = -1
+        #: int: the ID of the object on top of the object currently being carried (-1 if none)
         self.carrying_object_on_top_id = -1
+        #: int: the ID of the object the head is tracking to (-1 if none)
         self.head_tracking_object_id  = -1
+        #: int: the ID of the object that robot is localized to (-1 if none)
         self.localized_to_object_id = -1
+        #: int: the timestamp for the last image seen (>= 0)
```

Comment:
This timestamp is another millisecond timestamp relative to the boot time of the robot, correct?  along the lines of `last_observed_robot_timestamp` and `last_tapped_robot_timestamp` in `object.py`

We should probably clarify the docs for those types to stay how they're generated.. would be nice if this one was also called `last_image_robot_timestamp` as well to be consistent though i guess that's a breaking change at this point